src/routes/index.js

```diff
@@ -4,8 +4,9 @@ import crudBusRoutes from './api/crudBusRoutes';
 import userRoutes from './api/userRoutes';
 import assignDriver from './api/assignDriverRoutes';
 import unassigned from './api/unassignedRoute';
+import profileRoutes from './api/profileRoutes';
 const routes = express.Router();
```

Comment:
@fab-ryan, it is best practice to space the imports and other variable declarations with an empty line, just like variable declarations and the body of the file, then the body and the exports. It makes the code more readable and is aesthetically the right thing to do